www-src/app/replicator/cache/android_calendar_cache.coffee

```diff
@@ -0,0 +1,32 @@
+AndroidCalendarHandler = require "../../lib/android_calendar_handler"
+log = require("../../lib/persistent_log")
+    prefix: "AndroidCalendarCache"
+    date: true
+
+module.exports = class AndroidCalendarCache
```

Comment:
I would put a cache functionality in android_calendar_handler, instea of this snapshot cache ; which looks a very dangerous tool to me (although its simple and clean).